bedrock/firefox/templates/firefox/products/developer-quantum.html

```diff
@@ -7,7 +7,13 @@
 {% extends "firefox/base-pebbles.html" %}
 
 {% block page_title %}{{_('Firefox Developer Edition')}}{% endblock %}
-{% block page_desc %}{{ _('Built for those who build the Web. The only browser made for developers like you.') }}{% endblock %}
+{% block page_desc %}
+{% if l10n_has_tag('dev_edition_quantum_updates') %}
+  {{ _('Firefox Developer Edition is the blazing fast browser that offers cutting edge developer tools and the latest features like CSS Grid support and framework debugging.') }}
```

Comment:
Can we add this without a period? We just exposed that string as part of whatsnew/firstrun, and we could avoid the tag.

Reply:
Oh that's good news. Period and tag removed.